lib/gui/components/update-notifier/styles/_update-notifier.scss

```diff
@@ -20,3 +20,7 @@
   margin-bottom: 0;
 }
 
+.modal-update-notifier .modal-body a {
```

Comment:
Can you double check if we're using anchors in any other modal? I believe all modal anchors will benefit from this style, so we could move it to `modal/styles/_modal.scss` unless it conflicts with anything else. 

As a side note, have you checked how this behaves on the multiple states an anchor can be at any given time (e.g: `:link`, `:active`, `:focus`, etc)?